lib/danger/commands/local.rb

```diff
@@ -66,12 +66,15 @@ def run
 
     private
 
+    # this method is a duplicate of Commands::PR#configure_octokit
+    # - worth a refactor sometime?
```

Comment:
iirc I think Commands::PR was deprecated a while ago in favor of Commands::Local anyway, so I'm assuming `danger pr` should disappear and be removed completely one day even?

Reply:
I think it's the other way round - from the danger help:

```
    + local                             Run the Dangerfile locally. This command is
                                        generally deprecated in favor of `danger pr`.
```

Reply:
Ah right yep I need some sleep, it's the other way around indeed 👍

still, probably why it's a copy/paste here as local is eventually planned to be removed entirely one day 🙂